src/_includes/header.njk

```diff
@@ -49,7 +48,7 @@
 	{% if templateClass == "template-spotlight" %}
 		{% set hero = heroes.spotlight %}
 		{% include "heroes/primary-nav.njk" %}
-	{% endif %}
+	{% endif %} #}
```

Comment:
I think we can safely remove this comment section fully from the code, since we don't seem to need that anymore.